src/Blocks/GutenbergBlock.php

```diff
@@ -0,0 +1,71 @@
+<?php
+namespace BusinessProfileRender\Blocks;
+
+/**
+ * Class GutenbergBlock
+ */
+class GutenbergBlock {
+    public static function init() {
+        add_action( 'init', array( __CLASS__, 'register_block' ) );
+    }
+
+    public static function register_block() {
+
+        // Register the block script
+        wp_register_script(
+            'business-profile-render-gutenberg-block',
+            plugin_dir_url( __FILE__ ) . '../../build/gutenberg-block.js',
+            array('wp-blocks', 'wp-element', 'wp-components'),
+            filemtime( plugin_dir_path( __FILE__ ) . '../../build/gutenberg-block.js' ),
+            true
+        );
+
+        // Fetch JSON data and pass it to the block script
+        $business_profile_data = get_option('bpr_business_profile');
```

Comment:
Check JSON data validation, Handle the error if data does not exist.

Reply:
Empty array check added